libs/domains/services/feature/src/lib/hooks/use-deployment-history/use-deployment-history.ts

```diff
@@ -0,0 +1,18 @@
+import { useQuery } from '@tanstack/react-query'
+import { type ServiceType } from '@qovery/domains/services/data-access'
+import { queries } from '@qovery/state/util-queries'
+
+export interface UseDeploymentHistoryProps {
+  serviceId?: string
```

Comment:
We have a mix between optional and required id, should be required everywhere no ?